compiler/optimizations/gpuTransforms.cpp

```diff
@@ -169,6 +171,8 @@ class GpuizableLoop {
   std::vector<Symbol*> lowerBounds_;
   bool shouldErrorIfNotGpuizable_;
 
+  std::vector<CallExpr*> gpuAssertions_; // should this be a single CallExpr* ?
```

Comment:
You can remove this declaration of `gpuAssertions_` now, right? I don't see it used anywhere if I ctrl+F in the diff.